polyfills/Symbol/config.json

```diff
@@ -8,7 +8,7 @@
 		"safari": "<9",
 		"chrome": "< 50",
 		"firefox": "< 40",
-		"ios_saf": "<9",
+		"ios_saf": "<10",
```

Comment:
This is weird because the RUM suggests it was falsely served to iOS 10 before:

```
"Symbol","ios_saf","10.0.0","100","280","4","493","Yes","False positive - remove targeting"
```